### DISCUSSION TITLE
DB migration failed

### DISCUSSION THREAD

**Discussion by GrygorievEvgeniy** [chatwoot/chatwoot#7771] [Self hosted] [answered]
Caused by:
NoMethodError: undefined method `locked=' for #<InstallationConfig id: nil, name: "INSTALLATION_NAME", serialized_value: {"value"=>"Chatwoot"}, created_at: nil, u>
/home/chatwoot/chatwoot/lib/config_loader.rb:64:in `save_as_new_config'
/home/chatwoot/chatwoot/lib/config_loader.rb:52:in `save_general_config'
/home/chatwoot/chatwoot/lib/config_loader.rb:43:in `block in reconcile_general_config'
/home/chatwoot/chatwoot/lib/config_loader.rb:40:in `each'
/home/chatwoot/chatwoot/lib/config_loader.rb:40:in `reconcile_general_config'
/home/chatwoot/chatwoot/lib/config_loader.rb:23:in `process'
/home/chatwoot/chatwoot/db/migrate/20200510112339_create_installation_config.rb:11:in `change'
/home/chatwoot/chatwoot/lib/tasks/db_enhancements.rake:26:in `block (3 levels) in <main>'
/home/chatwoot/chatwoot/lib/tasks/db_enhancements.rake:18:in `each'
/home/chatwoot/chatwoot/lib/tasks/db_enhancements.rake:18:in `block (2 levels) in <main>'
Tasks: TOP => db:migrate
(See full trace by running task with --trace)

**Comment by rohitkush1** [comment]
Hi are you able to resolved the issue . i am also getting the same error.

**Reply by GrygorievEvgeniy** [reply]
Hi ,  I forgot what helps me.

**Comment by sojan-official** [comment, marked as answer]
https://github.com/chatwoot/chatwoot/issues/2926#issuecomment-909116375

during initial set up don't run db:migrate as code for some of the underlying migrations might have changed. load the schema from schema.rb
